test/test_cpp_standards.py

```diff
@@ -35,28 +114,48 @@ def test_cpp14():
     assert set(actual_wheels) == set(expected_wheels)
 
 
-def test_cpp17():
+cpp17_project = cpp_template_project.copy()
```

Comment:
I think I prefer this to making the copy etc inside the test functions like in `test_cpp11` and `test_cpp14`. At any rate, we should probably have a single way of doing this.

Reply:
Yes, good point. I agree.